app/marketing/signals.py

```diff
@@ -24,7 +24,7 @@
 from marketing.models import EmailSubscriber
 
 
-@receiver(post_save, sender=EmailSubscriber)
```

Comment:
why are we removing this ?

Reply:
Removing because the email drip was old. It said Hey Staff memeber, be sure to sign up for Slack!     We already started a new campaign that will send from acquia. 

As long as this isn't sending anymore, I am happy. Doesnt matter if deleted or archived.